go/pkg/log/server/property_test.go

```diff
@@ -290,6 +306,9 @@ func (suite *LogServerTestSuite) TestRecordLogDb_PushLogs() {
 		if _, ok := collections[position]; !ok {
 			collections[position] = types.NewUniqueID()
 		}
+		if position%2 == 0 {
+			suite.sysDb.AddCollection(ctx, collections[position].String())
```

Comment:
is this a way to make some of the collections appear as deleted?

Reply:
Will address in a follow up PR.

Reply:
I was just asking a question